webapp/src/editortoolbar.tsx

```diff
@@ -328,6 +331,83 @@ export class EditorToolbar extends data.Component<ISettingsProps, {}> {
     }
 }
 
+interface ZoomSliderProps extends ISettingsProps {
+    view: string;
+}
+
+interface ZoomSliderState {
+    zoomValue: number;
+}
+
+export class ZoomSlider extends data.Component<ZoomSliderProps, ZoomSliderState> {
+    private zoomMin = 0;
```

Comment:
might be useful if these were optional props with default values of 0 and 5 so the component is configurable